packages/aws-cdk-lib/aws-lambda/test/function-url.test.ts

```diff
@@ -231,11 +231,17 @@ describe('FunctionUrl', () => {
     });
 
     // WHEN
-    fn.addFunctionUrl({
+    const fnUrl = fn.addFunctionUrl({
       authType: lambda.FunctionUrlAuthType.NONE,
       invokeMode: lambda.InvokeMode.BUFFERED,
     });
 
+    if (fnUrl.authType === lambda.FunctionUrlAuthType.NONE) {
+      ; // ok
+    } else {
+      throw new Error('AuthType must be NONE');
+    }
```

Comment:
I think it is better to use jest `expect()`.

```suggestion
expect(fnUrl.authType).toBe(lambda.FunctionUrlAuthType.NONE);
```

Reply:
Agreed. Also, I believe this PR should be a `fix` not a `chore`

Reply:
Since there is no change in the resource being created, I don't think this fix needs an integ test, but if the maintainer thinks it does, I'll change it to a fix.